components/Selector.jsx

```diff
@@ -9,45 +9,12 @@ import { faTwitter } from '@fortawesome/free-brands-svg-icons';
 import '@fortawesome/fontawesome-svg-core/styles.css';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import PulseSvg from '@components/PulseSvg';
-
-function useFetch(searchStr, resourceType = 'supply', maxResults = 25) {
-    const url = `https://covidconnect.vercel.app/api/1.1/data?city=${searchStr}&resource_type=${resourceType}&max_results=${maxResults}`;
-    const [data, setData] = useState([]);
-    const [loading, setLoading] = useState(true);
-
-    useEffect(() => {
-        async function fetchUrl() {
-            if (!searchStr.length) {
-                return;
-            }
-            const response = await fetch(url);
-            const temp = await response.json();
-            const json = {
-                res: temp
-            };
-            if (
-                !json ||
-                !json.res ||
-                !json.res.response ||
-                !json.res.response.api_response ||
-                !json.res.response.api_response.statuses
-            ) {
-                setData([]);
-            } else {
-                setData(json.res.response.api_response.statuses);
-            }
-            setLoading(false);
-        }
-        fetchUrl();
-    }, [url]);
-    return [data, loading];
-}
-export { useFetch };
+import useFetch from '@hooks/use-fetch';
 
 const Selector = ({ data, page, placeholder, localeState, localeDistrict }) => {
     const [searchStr, setSearchStr] = useState('');
     const [editing, setEditing] = useState(false);
-    const [covidConnectResults, loading] = useFetch(searchStr);
+    const [covidConnectResults, loading] = useFetch({ city: searchStr });
```

Comment:
so the home page shows all materials, and the individual page will show based on the dropdown?

Reply:
No the behavior will stay the same, I have just changed the function call method